src/main/webapp/project-add-user-form.jsp

```diff
@@ -0,0 +1,27 @@
+<%@ page import="com.rtb.projectmanagementtool.project.*" %>
+
+<%
+    Long userId = (Long) request.getAttribute("userId");
+    ProjectData project = (ProjectData) request.getAttribute("project");
+%>
+
+<form action="/add-user-to-project" method="POST">
```

Comment:
Would be cool if the user can do multiple operations (add multiple users + create tasks) without having to reload the page every time.  Think about a professor having to create a project for 30 students, each student having 1 task each.  That'll be 60 page loads at least